movieDetail.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Movie Box</title>
+    <link rel="stylesheet" href="./assests/styleSheets/movieDetail.css" />
+  </head>
+  <body>
+    <a href="./index.html" class="header-text"><h1>Movies</h1></a>
+    <figure class="main-container" id="movie-detail">
+      <img
+      src="./assests/images/luca-cover.jpeg"
+      alt="Movie cover image" class="cover-image" />
+      <figcaption class="detail-text">
+        <section class="detail-text-header">
+          <h2 class="movie-title">Luca</h2>
+          <p class="movie-rate">7/10</p>
+        </section>
+        <summary class="movie-description">
```

Comment:
`summary` has to be a child of `details`. I suggest changing it to `p` and moving it inside the section.

Reply:
thanksssss